server/controllers/jobRegistration.controller.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+const responseUtil = require('../utils/responseUtils');
+const MessageUtil = require('../utils/messageUtil');
+const jobRegService = require("../services/userJobRegister.service");
+const { v4: uuidv4 } = require('uuid');
+
+exports.getAllJobRegistrations = async (req, res) => {
```

Comment:
Currently for courseRegistration, Only Create Operation out of CRED is implement, We need to confirm if we require Update CourseRegistration and will it resend a email to the candidate.

Reply:
I too think `create` operation is only needed for now, before deployment we will have a cleanup where we can remove unwanted/unused methods.